examples/zkapps/05-common-types-and-functions/src/main.ts

```diff
@@ -26,301 +25,295 @@ import {
 import { LedgerContract } from './LedgerContract.js'
 import { BasicMerkleTreeContract } from './BasicMerkleTreeContract.js'
 
-async function main() {
-  await isReady;
+await isReady;
 
-  // --------------------------------------
+// --------------------------------------
 
-  const num1 = UInt32.from(40);
-  const num2 = UInt64.from(40);
+const num1 = UInt32.from(40);
+const num2 = UInt64.from(40);
 
-  const num1EqualsNum2: Bool = num1.toUInt64().equals(num2);
-  
-  console.log("// --------------------------------------");
-  console.log(`num1 == num2: ${num1EqualsNum2.toString()}`);
-  console.log(`Fields in num1: ${num1.toFields().length}`);
+const num1EqualsNum2: Bool = num1.toUInt64().equals(num2);
 
-  // --------------------------------------
+console.log("// --------------------------------------");
+console.log(`num1 == num2: ${num1EqualsNum2.toString()}`);
```

Comment:
nit with the entire tutorial: it uses
```ts
`some text: ${something.toString()}`
```
even though putting stuff inside string templates `toString`s them already, so we could just use 
```ts
`some text: ${something}`
```

but you don't have to change that now!

Reply:
Thanks! I think a little elegance goes a long way in example code so I'm in favour of making this change. Not all of them can be coerced automatically, however (e.g. bools). In those cases I've moved the `.toString()` calls to the const declaration so we've still managed to strip them from string templates as suggested.